src/Sylius/Behat/Context/Api/Shop/PaymentRequestContext.php

```diff
@@ -81,4 +92,33 @@ public function putPaymentRequest(string $paymentRequestUri, array $payload = []
 
         $this->client->executeCustomRequest($request);
     }
+
+    /**
+     * @Then my payment request with action :action for payment method :paymentMethod should have state :state
+     */
+    public function myPaymentRequestShouldBeCancelled(string $action, PaymentMethodInterface $paymentMethod, string $state): void
+    {
+        if ($this->getRequestForPaymentRequestWithAction($action) === null) {
+            throw new \InvalidArgumentException(sprintf('Payment request with action "%s" not found', $action));
+        }
+
+        $request = $this->getRequestForPaymentRequestWithAction($action);
+        $this->client->executeCustomRequest($request);
+        $response = $this->client->getLastResponse();
+
+        Assert::same($this->responseChecker->getValue($response, 'action'), $action, sprintf('Payment request should have action %s', $action));
+        Assert::true(str_contains($this->responseChecker->getValue($response, 'method'), $paymentMethod->getCode()), sprintf('Payment request should be for %s payment method', $paymentMethod->getCode()));
```

Comment:
```suggestion
        Assert::true(str_contains($this->responseChecker->getValue($response, 'method'), $paymentMethod->getCode()), sprintf('Payment request should have payment method %s', $paymentMethod->getCode()));
```
To keep it consistent with the rest. Also, fix the formatting of these.